main.py

```diff
@@ -79,23 +73,23 @@ def process_season(series_id):
                 #file_conv = file
                 #title_conv = title
 
-                if scrub__and__strings:
+                if SCRUB__AND__STRINGS:
                     file_conv = file_conv.replace('and','')
                     title_conv = title_conv.replace('and','')
                 
-                if allow_multi_part_episode_files and is_multi_episode(str.lower(file)):
+                if ALLOW_MULIT_PART_EPISODE_FILES and is_multi_episode(str.lower(file)):
                     title_conv = title_conv[:-1] # remove the part ID from the end of the name (eg 'Epidode Name (1)' becomes 'Episode Name')
                     title_conv = title_conv.replace('part','') # remove the string 'part' from the file name (eg 'Episode Name Part 1' becomes 'Episode Name' (as the 1 was removed above) )
                     # note: If a file used something other that 'PART n' this won't pick it up
                     # Also, 'part' might be a valid part of the main episode name, so this would break that - should fix to only remove 'part' from the end
                     # But this only gets applied where we already know it's a multi episode file, so shouldn't cause too many issues
 
-                if default_episode(title) and ignore_default_episdode_name:
+                if default_episode(title) and IGNORE_DEFAULT_EPISODE_NAME:
                     # This function tests is the episode title is just 'default' naming. eg 'Episode 1'
                     # In these cases, the user might not want/need 'Episode 1' in the file name.
                     # What I'd like to do is then test the S01E01 matches, but for now, we'll just test for it with the option to skip these files
 
-                    1==1
+                    pass
```

Comment:
`pass` is meant for these exact situations where you don't actually want to do anything. Its a no-op/null operation.

Reply:
Thanks for the tip on using pass - exactly what I needed!